client/send-text.cc

```diff
@@ -42,19 +42,21 @@ static int usage(const char *progname) {
             "\t-o              : Only run once, don't scroll forever.\n"
             "\t-c<RRGGBB>      : Text color as hex (default: FFFFFF)\n"
             "\t-b<RRGGBB>      : Background color as hex (default: 000000)\n"
+            "\t-r              : Scroll text vertically (rotate)\n"
```

Comment:
see above, fixed in https://github.com/miloh/flaschen-taschen/commit/0fbb88a7ed1c112ebe2a46f771b8109c74699d5c